src/server/redis_connection.h

```diff
@@ -45,6 +45,7 @@ class Connection : public EvbufCallbackBase<Connection> {
     kCloseAfterReply = 1 << 6,
     kCloseAsync = 1 << 7,
     kMultiExec = 1 << 8,
+    KReadWrite = 1 << 9,
```

Comment:
The default state of the flag be disabled? Should it be `KReadOnly` here?

Reply:
It seems that the original logic is `KReadOnly` ==1, and the default flag is 0, so I choose `KReadWrite` here, which means the default option is `KReadWrite` ==0 (`KReadOnly` ==1).

Reply:
What is the logic for Redis? The original logic may be wrong.
We need to align with Redis, not with the original.

Reply:
https://redis.io/commands/readonly/
> Normally replica nodes will redirect clients to the authoritative master for the hash slot involved in a given command, however clients can use replicas in order to scale reads using the READONLY command.

I see, Redis default option is `READWRTIE`, will change it .